go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/smartcontractkit/chainlink-starknet v0.0.0-20220704110337-2b93a45d825d
 	github.com/smartcontractkit/chainlink-terra v0.1.4-0.20220620174806-cd472d246b36
 	github.com/smartcontractkit/libocr v0.0.0-20220701150323-d815c8d0eab8
-	github.com/smartcontractkit/ocr2vrf v0.0.0-20220712214142-44a5f8469a02
+	github.com/smartcontractkit/ocr2vrf v0.0.0-20220718193930-3449bbd93a77
```

Comment:
I think you need to submit changes to `go.sum`, looks like they're missing from the diff.